zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/NamePrefixThreadFactory.java

```diff
@@ -21,19 +21,19 @@
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.atomic.AtomicLong;
 
-public class SchedulerThreadFactory implements ThreadFactory {
+public class NamePrefixThreadFactory implements ThreadFactory {
```

Comment:
let's call it `NamedThreadFactory`, it's kind of a common name

PS: you will get thousands of results if you search "NamedThreadFactory" on the entire GitHub site.

Reply:
BTW, it could be reused in many places, do we have a more generic place to locate this class?